actions/extractor/tools/autobuild-impl.ps1

```diff
@@ -1,21 +1,28 @@
-if (($null -ne $env:LGTM_INDEX_INCLUDE) -or ($null -ne $env:LGTM_INDEX_EXCLUDE) -or ($null -ne $env:LGTM_INDEX_FILTERS)) {
-    Write-Output 'Path filters set. Passing them through to the JavaScript extractor.' 
-} else {
-    Write-Output 'No path filters set. Using the default filters.'
-    # Note: We're adding the `reusable_workflows` subdirectories to proactively
-    # record workflows that were called cross-repo, check them out locally,
-    # and enable an interprocedural analysis across the workflow files.
-    # These workflows follow the convention `.github/reusable_workflows/<nwo>/*.ya?ml`
-    $DefaultPathFilters = @(
-        'exclude:**/*',
-        'include:.github/workflows/*.yml',
-        'include:.github/workflows/*.yaml',
-        'include:.github/reusable_workflows/**/*.yml',
-        'include:.github/reusable_workflows/**/*.yaml',
-        'include:**/action.yml',
-        'include:**/action.yaml'
-    )
+# Note: We're adding the `reusable_workflows` subdirectories to proactively
+# record workflows that were called cross-repo, check them out locally,
+# and enable an interprocedural analysis across the workflow files.
+# These workflows follow the convention `.github/reusable_workflows/<nwo>/*.ya?ml`
+$DefaultPathFilters = @(
+    'exclude:**/*',
+    'include:.github/workflows/*.yml',
+    'include:.github/workflows/*.yaml',
+    'include:.github/reusable_workflows/**/*.yml',
+    'include:.github/reusable_workflows/**/*.yaml',
+    'include:**/action.yml',
+    'include:**/action.yaml'
+)
 
+if ($null -ne $env:LGTM_INDEX_FILTERS) {
```

Comment:
If `$env:LGTM_INDEX_FILTERS` is an empty string, does this still work?

Reply:
Also, `LGTM_INDEX_FILTERS` includes both the include and exclude filters?

Reply:
> Also, LGTM_INDEX_FILTERS includes both the include and exclude filters?

Yes - `LGTM_INDEX_FILTERS` is a string containing `include:<pattern>` and `exclude:<pattern>` substrings separated by newlines. https://github.com/github/codeql/blob/7ed8a85e0845856442f004e95808db9ba485f41f/javascript/extractor/src/com/semmle/js/extractor/AutoBuild.java#L93 explains it.

Reply:
> If $env:LGTM_INDEX_FILTERS is an empty string, does this still work?

`$null -ne $env:LGTM_INDEX_FILTERS` is `True` in this case, and then we concatenate the default filters with an extra newline at the end, so it still works.